safe_transaction_service/history/services/collectibles_service.py

```diff
@@ -363,7 +387,49 @@ def get_collectibles_with_metadata(
                     metadata,
                 )
             )
-        return collectibles_with_metadata
+        return collectibles_with_metadata, count
+
+    def get_collectibles_with_metadata_v1(
```

Comment:
I think `get_collectibles_with_metadata` and `get_collectibles_with_metadata_paginated` are better than `v1` and `v2`. Remember that the service doesn't need to know about what we are doing on the views 😉

Reply:
Yep, seems more clear, I will check.